templates/example_MPH_plots.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    target = 'CD8'
</pre><p>We're most often going to be computing the MPH for specific mass windows, not specific targets on the panel. It would be more useful to take in mass_start and mass_stop, instead of the name of the channel. The initial notebook I sent you had that as an option. You should be able to use that to see how you need to modify the bin_files.get_median_pulse_height to take in an arbitrary mass window</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>